system_tests/attempt_system_tests.py

```diff
@@ -136,11 +139,49 @@ def prepare_to_run():
     decrypt_keyfile()
 
 
+def get_parser():
+    """Get an argument parser to determine a list of packages."""
+    parser = argparse.ArgumentParser(
+        description='google-cloud tests runner.')
+    help_msg = ('List of packages to be tested. '
+                'If left blank, tests all packages.')
+    parser.add_argument('packages', nargs='*',
+                        default=ALL_MODULES, help=help_msg)
+    return parser
+
+
+def get_modules():
+    """Get the list of modules names to run system tests for."""
+    parser = get_parser()
+    args = parser.parse_args()
+    if args.packages is ALL_MODULES:
+        result = list(MODULES)
+        if sys.version_info[:2] != (2, 7):
+            result.remove(BIGTABLE_API)
```

Comment:
It is indeed a fossil. As is the warning and the conditional deps in `setup.py`?